<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>21</y>
     <width>778</width>
     <height>548</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_strategy">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>查询条件</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_strategy">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>实时涨幅大于等于X，小于Y：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>X = </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_todayUp">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-12.000000000000000</double>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Y = </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_todayDown">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-12.000000000000000</double>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>昨日涨幅大于等于X，小于Y：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>X = </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_yestodayUp">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-12.000000000000000</double>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Y = </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_yestodayDown">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="correctionMode">
                <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-12.000000000000000</double>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkBox_1">
             <property name="text">
              <string>前M个交易日不出现连续N天上涨</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>M = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="checkBox1_M"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>N = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="checkBox1_N"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>前M个交易日不出现连续N天上涨均超过X</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>M = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="checkBox2_M"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>N = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="checkBox2_N"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>X = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="checkBox2_X">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-12.000000000000000</double>
             </property>
             <property name="maximum">
              <double>12.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>前M个交易日整体涨幅小于X</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>M = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="checkBox3_M"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>X = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="checkBox3_X">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-12.000000000000000</double>
             </property>
             <property name="maximum">
              <double>12.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>当日振幅小于</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_swing">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-12.000000000000000</double>
             </property>
             <property name="maximum">
              <double>12.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>不重复记录已出现在结果中的股票</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>量比大于等于</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="lbValue">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-99.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QCheckBox" name="checkBox_9">
             <property name="text">
              <string>当前价小于第前M日收盘价</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>M = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="checkBox9_M">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>15</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QCheckBox" name="checkBox_7">
             <property name="text">
              <string>实时换手率大于</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hslValue">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-99.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QCheckBox" name="checkBox_10">
             <property name="text">
              <string>连续n日换手率大于x</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>n = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hsl10n"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>x = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hsl10x"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QCheckBox" name="checkBox_11">
             <property name="text">
              <string>连续n日换手率大于前5日平均值x倍</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>n = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hsl11n"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>x = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hsl11x"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView_result">
      <property name="minimumSize">
       <size>
        <width>561</width>
        <height>150</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="text">
         <string>刷新</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_recorve">
        <property name="text">
         <string>恢复设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_boughtRecord">
        <property name="text">
         <string>购买记录</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_downLoad">
        <property name="text">
         <string>下载</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>开始</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
